modules/browsiRtdProvider.js

```diff
@@ -283,36 +248,37 @@ export const browsiSubmodule = {
   /**
    * get data and send back to realTimeData module
    * @function
-   * @param {adUnit[]} adUnits
-   * @param {function} onDone
+   * @param {string[]} adUnitsCodes
    */
-  getData: sendDataToModule,
-  init: init
+  getTargetingData: sendDataToModule,
+  init: init,
 };
 
-function init(config, gdpr, usp) {
+function init() {
   return true;
 }
 
-export function beforeInit(config) {
+function beforeInit(config) {
   const confListener = config.getConfig(MODULE_NAME, ({realTimeData}) => {
-    try {
-      _moduleParams = realTimeData.dataProviders && realTimeData.dataProviders.filter(
-        pr => pr.name && pr.name.toLowerCase() === 'browsi')[0].params;
-      confListener();
-      _moduleParams.auctionDelay = realTimeData.auctionDelay;
-      _moduleParams.timeout = realTimeData.timeout || DEF_TIMEOUT;
-    } catch (e) {
-      _moduleParams = {};
-    }
-    if (_moduleParams.siteKey && _moduleParams.pubKey && _moduleParams.url) {
+    setModuleData(realTimeData);
+    confListener();
+    if (_moduleParams && _moduleParams.siteKey && _moduleParams.pubKey && _moduleParams.url) {
       collectData();
     } else {
       utils.logError('missing params for Browsi provider');
     }
   });
 }
 
+export function setModuleData(realTimeData) {
+  try {
+    _moduleParams = realTimeData.dataProviders && realTimeData.dataProviders.filter(
+      pr => pr.name && pr.name.toLowerCase() === 'browsi')[0].params;
```

Comment:
Yeah this should be passed to you in the core util passbacks, then you can run a `setModuleData`